python/uv/private/lock.bzl

```
@@ -327,10 +327,15 @@ def _maybe_file(path):
 def _expand_template_impl(ctx):
     pkg = ctx.label.package
     update_src = ctx.actions.declare_file(ctx.attr.update_target + ".py")
+
+    # Fix the path construction to avoid absolute paths
+    # If package is empty (root), don't add a leading slash
+    dst = ctx.attr.output if pkg == "" else "{}/{}".format(pkg, ctx.attr.output)
```

```suggestion
    dst = "{}/{}".format(pkg, ctx.attr.output) if pkg else ctx.attr.output
```

This is a little bit shorter. I think the `lock` file rule has not been released yet, so not updating CHANGELOG is fine here.